packages/calypso-build/babel.config.js

```diff
@@ -0,0 +1,71 @@
+/** @format */
+
+const isCalypsoClient = process.env.CALYPSO_CLIENT === 'true';
+const isBrowser = isCalypsoClient || 'true' === process.env.TARGET_BROWSER;
+
+const modules = isBrowser ? false : 'commonjs'; // Use commonjs for Node
+
+const targets = isBrowser
+	? { browsers: [ 'last 2 versions', 'Safari >= 10', 'iOS >= 10', 'ie >= 11' ] }
```

Comment:
Let's pick this change to here: https://github.com/Automattic/wp-calypso/pull/31561

Reply:
Fixed by rebasing